package.json

```diff
@@ -27,7 +27,7 @@
     "@marcj/marshal": "2.1.13",
     "@mojotech/json-type-validation": "3.1.0",
     "@sapphire/shapeshift": "3.6.0",
-    "@sinclair/typebox": "0.24.41",
+    "@sinclair/typebox": "^0.25.21",
```

Comment:
I think you need to use the exact `0.25.21` version without `^` because renovate bot deals with the updates (not sure how that all works but @moltar always says this and all other module versions are specific too).

Reply:
That's right. Renovate will auto-bump even patch versions as they land. So it is better to use an exact version.

Reply:
@hoeck @moltar Thanks! Updated to use exact version.